### ISSUE TITLE
hack-nights page name-change related cleanup

### ISSUE THREAD

**Issue by drubgrubby** [hackforla/website#748]
### Overview
We have changed ../website/hack-nights-page.html to ../website/hack-nights.html.  

We now have both: 
../_sass/components/hack-nights-page.scss 
and
../_sass/components/_hack-nights.scss

It appears that some of the tags from hack-nights.scss are used on pages other than hack-nights pages. 

We also have both:
../website/hack-nights.html
and
../website/_includes/hack-nights.html

### Action Items
These files need the Code Zamboni(tm) - someone to go over these pages and clean them up.  We should get rid of resources that are no longer in use and consolidate the remaining resources. 

My suggestion:
1) See what styles on _hack-nights.scss are still in use and move them to the appropriate .scss page (most likely hack-nights-page.scss
2) Once all of the styles on _hack-nights.scss are accounted for, delete that page.
3) Rename hack-nights-page.scss to _hack-nights.scss

As far as the duplicate, hack-nights.html pages... I'm not sure that ../website/_includes/hack-nights.html is still in use.  If not we can delete this page.  If it is, determine if there is a problem with two pages having the same name (it seems like a bad idea to me), and, if yes, take appropriate action.

**Comment by ExperimentsInHonesty** [comment]
change the page name to events.html

**Comment by drubgrubby** [comment]
This is complete and a Pull has been Requested.

This main part of this was renaming hack-nights.html to events.html.

In addition, there was some cleanup to related files:

Add underscores to .scss files for consistancy
Remove undescore from _main.scss 
(cnk suggested the above two changes to keep naming convention consistent)
Change hack-nights-page.scss to _events.scss
Delete ../website/_includes/hack-nights.html
Moved a couple of styles from hack-nights.scss to events.scss

**Comment by ExperimentsInHonesty** [comment]
Page still has issues on mobile I am reopening ![image](https://user-images.githubusercontent.com/37763229/94770247-20736f80-0382-11eb-9fee-efab62b1c9bf.png)

**Comment by drubgrubby** [comment]
@ExperimentsInHonesty - I was going to do the work on the mobile site on issue #403, where all of the other work on the hack-nights/events page was done; this issue was intended to simply be a cleanup issue, then we added the name change at the last meeting (which also probably should have been added to the other issue). I'm happy to do the mobile work attached to this issue instead, but then we can probably close #403.  However, my inclination is to leave this issue close, and I will continue to work on the mobile stuff on the other issue. Or, we can make a new issue for the mobile fixes and close #403 also.
